### ISSUE TITLE
Error sending a message with an attachment

### ISSUE THREAD

**Issue by mredaelli** [matterhorn-chat/matterhorn#537]
```
An unexpected error has occurred! The exception encountered was:
  JSONDecodeException {jsonDecodeExceptionMsg = "Error in $['client_ids']: expected Seq a, encountered Null", jsonDecodeExceptionJSON = "{\"file_infos\":[{\"id\":\"j8xjher767b38rzjgg8tz9zeno\",\"user_id\":\"grnxskt6n7r4mpqxt1orpboq1y\",\"create_at\":1571845023615,\"update_at\":1571845023615,\"delete_at\":0,\"name\":\"missing_staging\",\"extension\":\"\",\"size\":3324,\"mime_type\":\"\"}],\"client_ids\":null}"}
Please report this error at https://github.com/matterhorn-chat/matterhorn/issues
```
```
❯ matterhorn --version
matterhorn 50200.2.0
 using mattermost-api 50200.1.2
```

**Comment by jtdaugherty** [comment]
Thanks for the report! This is interesting because this is a bug that as far as I can tell was fixed in `50200.2.0`, according to this changelog entry:
```
  Bug fixes:
   * Updated to mattermost-api 50200.1.2 which includes a bugfix for a
     server "UploadResponse" with a null `client_ids` field.  This
     manifested as an error report when attempting to post a message
     with an attachment to a general channel.
```
Are you able to upgrade Matterhorn to the latest version and see if you get better results?

**Comment by jtdaugherty** [comment]
(Closed due to inactivity. Please re-open if this problem is unresolved.)
